frontend/src/components/tools/FilterReportsByDate.vue

```diff
@@ -30,6 +30,24 @@
       <button class="button is-primary" @click="dateRange(2, 'years')">
         <span>{{ $t('last_2_years') }}</span>
       </button>
+      <button class="button is-primary" @click="
+          reported_at_min = new Date('2016-01-01')
+          reported_at_max = new Date('2016-12-31')
+        ">
+        <span>2016</span>
+      </button>
+      <button class="button is-primary" @click="
+          reported_at_min = new Date('2017-01-01')
+          reported_at_max = new Date('2017-12-31')
+        ">
+        <span>2017</span>
+      </button>
+      <button class="button is-primary" @click="
+          reported_at_min = new Date('2018-01-01')
+          reported_at_max = new Date('2018-12-31')
+        ">
+        <span>2018</span>
+      </button>
```

Comment:
Would it be possible to automate the creation of those button for the last 3 years?

We will need to go and change that in a year otherwise :D